docs/asciidoc/front_matter.adoc

```diff
@@ -11,16 +16,56 @@ The primary functionality of DRS is to map a logical ID to a means for physicall
 Each implementation of DRS can choose its own id scheme, as long as it follows these guidelines:
 
 * DRS IDs are strings made up of uppercase and lowercase letters, decimal digits, hypen, period, underscore and tilde [A-Za-z0-9.-_~]. See https://tools.ietf.org/html/rfc3986#section-2.3[RFC 3986 § 2.3].
-* Note to server implementors: internal IDs can contain other characters, but they MUST be encoded into valid DRS IDs whenever exposed by the API.   
+* DRS IDs can contain other characters, but they MUST be encoded into valid DRS IDs whenever exposed by the API.  This is because non-encoded IDs may interfere with the interpretation of the `objects/{id}/access` endpoint.  To overcome this limitation use percent-encoding of the ID, see https://tools.ietf.org/html/rfc3986#section-2.4[RFC 3986 § 2.4]
```

Comment:
A small clarification on language - would this be clearer if it says "MUST Be encoded into valid DRS IDs whenever exposed _to_ the API" instead of _by_? Since, in the event of other characters, clients will need to make sure proper percent-encoding is handled when passing the DRS ID to the `objects/{id}/access` endpoint in order to get a valid response.

Reply:
Thanks, I changed this to "whenever they are used in API calls"